.github/actions/assign-fixed-issues/entrypoint.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 set -e
 
-# 1. Find the issues that this PR 'fixes'.
+# 1. Proceed only when acting on an opened pull request.
+
+action=$(jq -r '.action' $GITHUB_EVENT_PATH)
+
+if [ "$action" != 'closed' ]; then
```

Comment:
This should be `'opened'`. The idea is that we assign any fixed issues to the author of a PR when that PR is opened.

Reply:
Yeah, that was a typo 😓